core/src/test/java/com/aem/demo/core/services/impl/RssFeedServiceImplTest.java

```diff
@@ -0,0 +1,17 @@
+package com.aem.demo.core.services.impl;
+
+
+import junit.framework.TestCase;
+import uk.org.lidalia.slf4jtest.TestLogger;
+import uk.org.lidalia.slf4jtest.TestLoggerFactory;
+
+public class RssFeedServiceImplTest extends TestCase {
+
+    private RssFeedServiceImpl rssFeedService = new RssFeedServiceImpl();
+
+    private TestLogger logger = TestLoggerFactory.getTestLogger(rssFeedService.getClass());
+
+    public void testReadFeed() {
+        rssFeedService.readFeed().forEach(System.out::println);
```

Comment:
It's hard to test something with real rss feed)
Try to use mockito here and mock the response from request